docs/development/issue-labeling.md

```diff
@@ -63,14 +63,18 @@ Add the `breaking` label for Issues or PRs which contain changes that are not ba
     priority-2-important
     priority-3-normal
     priority-4-low
+    priority-5-triage
 
 </details>
 
 Use these to assign a priority level to an issue.
+Incoming issues are labeled `priority-5-triage` by default, this label should be replaced with a proper priority (low/normal/important/critical).
 Make a best-effort attempt to select a proper priority.
 Nothing bad will happen if you select a "wrong" priority.
 At a high level: critical = needs immediate fix, important = to be prioritized ahead of others, normal = default priority, low = trivial issue, or impacts a very small % of the user base.
 
+Use [this search](https://github.com/renovatebot/renovate/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+-label%3Apriority-1-critical+-label%3Apriority-2-important+-label%3Apriority-3-normal+-label%3Apriority-4-low++-label%3Apriority-5-triage) to find any issues which are missing a priority label.
```

Comment:
Please check my work on the search query before approving again.

Reply:
Yes, let's looks right. Shall we bulk label those as `priority-5-triage`?

Reply:
Yeah let's bulk label those. I'll let you do the honors on that one. 😄

Reply:
I don't think you can actually bulk label those...
I don't see a checkbox before the issue to select it from the search screen.
So it seems bulk labeling is right out.

@rarkins Want me to label the issues manually then?

Reply:
Oh you probably need `maintainer` level rights to bulk label things... 😄

Reply:
![image](https://user-images.githubusercontent.com/1798109/109813647-eeff1880-7c2d-11eb-88ce-ce578dcfc022.png)
🙃

Reply:
Thank you @viceice for doing the bulk labeling. 😄 

I'm resolving this conversation.